src/helpers/distance.js

```diff
@@ -29,7 +29,7 @@ export function distanceNumber(a, b) {
 }
 
 export function distance(a) {
-  if(a) {
+  if (a) {
```

Comment:
If you install Prettier and set to format on save it could take care of this stuff for you automatically.